assets/training/forecasting_demand/environments/automl-gpu/context/conda_dependencies.yaml

```diff
@@ -5,24 +5,23 @@ channels:
 dependencies:
 - python=3.8
 - pip=22.1.2
-- pytorch=1.9.0
 - numpy~=1.22.3
-- scikit-learn=0.22.1
-- pandas~=1.1.5
+- scikit-learn=1.1.3
+- pandas~=1.3.5
 - py-xgboost=1.3.3
 - holidays=0.10.3
 - cudatoolkit=11.1
 - pyopenssl=23.2.0
 - psutil>=5.2.2,<6.0.0
-- GitPython=3.1.30
+- GitPython=3.1.32
 - scipy~=1.7.0
 - tqdm
 - setuptools=65.5.1
 - wheel=0.38.1
 - openssl=1.1.1s
 - pip:
   - inference-schema
-  - cryptography==41.0.0
+  - cryptography==41.0.3
```

Comment:
I don't think you need to have cryptography here

Reply:
Removed.